lib/msal-browser/src/config/Configuration.ts

```diff
@@ -14,6 +14,7 @@ export type BrowserAuthOptions = {
     clientId: string;
     authority?: string;
     knownAuthorities?: Array<string>;
+    instanceMetadata?: Array<IInstanceDiscoveryMetadata>;
```

Comment:
Naming here might get confusing when we implement the other perf changes which introduced authorityMetadata in msal-core. Open to suggestions for a better name here.

Reply:
In `msal-core`, we had `authorityMetadata` passed as a JSON string, instead of a deserialized object. This was done so that applications wouldn't have to parse the response, and could pass the JSON string as-is (and should be considered opaque).

Reply:
And agree about the naming. Not sure on a better suggestion at the moment.

Reply:
Ahh gotcha. That's a good suggestion, will try to do something similar. w.r.t. naming, how about `cloudDiscoveryMetadata` ?

Reply:
Updated name and input type to string